File: Python developer/step07/7.03/7.3.01.py
'''

Вам дана последовательность чисел заканчивающаяся -1. Выведете ее в обратном порядке без учета -1.

Ввод:

Последовательность чисел

Вывод:

Последовательность чисел в обратном порядке.

Sample Input 1:

1
2
3
4
-1

Sample Output 1:

4
3
2
1

Sample Input 2:

1
1
1
-1

Sample Output 2:

1
1
1

Sample Input 3:

-1

Sample Output 3:

Sample Input 4:

1
2
2
3
3
4
4
5
-1

Sample Output 4:

5
4
4
3
3
2
2
1

'''

stack = []
element = int(input())

while element != -1:
    stack.append(element)
    element = int(input())

for i in reversed(stack):
    print(i)

File: Diving into JavaScript: For Beginners/step07/7.03/7.3.01.js
/*

Перед вами программа, которая запрашивает у пользователя строку, а затем выводит каждый символ этой строки на отдельной строке в нижнем регистре. 
Вставьте вместо многоточия верное выражение.

Sample Input 1:

Программа

Sample Output 1:

п
р
о
г
р
а
м
м
а

Sample Input 2:

Тесто

Sample Output 2:

т
е
с
т
о

*/

const sentence = prompt();
for (let i = 0; i < sentence.length; i++) {
  console.log(sentence[i].toLowerCase());
}
